main.go

```diff
@@ -244,6 +248,26 @@ func process(plan types.Plan) error {
 	return nil
 }
 
+func helmRepoUpdate() error {
```

Comment:
This looks spot on 👍 

Can we try doing an exec with the shell script added?

Reply:
Done